central/sac/root_scope_checker_core_context_enricher.go

```diff
@@ -130,53 +82,6 @@ func (se *Enricher) GetPreAuthContextEnricher(authzTraceSink observe.AuthzTraceS
 	}
 }
 
-func (se *Enricher) cacheForClient(client sacClient.Client) expiringcache.Cache {
```

Comment:
It would be worth checking whether the client API defined in `pkg/sac/client` is used anywhere else than in the code being removed here.